promotions/lib/controllers/backend/solidus_promotions/admin/benefits_controller.rb

```diff
@@ -14,7 +14,7 @@ def new
       def create
         @benefit = @benefit_type.new(benefit_params)
         @benefit.promotion = @promotion
-        if @benefit.save(validate: false)
+        if @benefit.save
```

Comment:
That `validate: false` seems strange to have been there in the first place.  Do we know if it was intentional?  and if so, does removing it cause other problems when the save could now fail because of validation?  At a minimum, I wonder if there needs to be a `status: :unprocessable_entity` added to the render for Turbo.  I think the same goes for the update action on line 42 below.. `render :edit, layout: false, status: :unprocessable_entity`

Reply:
So... I wrote this code, initially to get a basic UI working. It turned out somewhat better - but still very similar - to the legacy promotion admin, and I'm honestly not sure why I introduced that. At this point, we would like to move towards the new, and for that we have to re-write all of this anyways...

Reply:
Good catch @boomer196 !